specs/UpdateRuntime.md

```diff
@@ -0,0 +1,229 @@
+# Background
+The new version of an app might require a newer version of Edge WebView2 Runtime.
+
+An app updater may wish to ensure that a particular minimum version of the Edge WebView2 Runtime is installed before upgrading the app to a version that requires those features.
+
+Alternatively, the newer version of the app could use feature detection to disable portions of the app that rely on new WebView2 Runtime features. In this alternate scenario, the app updater would install the new version of the app immediately, and then request that the Edge WebView2 Runtime be updated, so that the updated app can start taking advantage of the new features.
+
+Edge WebView2 Runtime is auto updated and normally the latest version should be already installed. However, there could be cases that we need trigger Edge WebView2 Runtime
+update to ensure coordinated app and WebView2 Runtime update.
+
+# Description
+You may call the `UpdateRuntime` API to check and install updates to installed Edge WebView2 Runtime. This is useful when the app wants to coordinate app and
+Edge WebView2 Runtime update.
+
+# Examples
+## .NET, WinRT
+```c#
+using Microsoft.Web.WebView2.Core;
+async protected bool EnsureWebView2RuntimeVersion(string minimalVersionRequired)
```

Comment:
Use Task<bool> for async methods

Reply:
No need for a Task. The result is throw away anyway.